pkg/process/net/common_linux.go

```diff
@@ -15,15 +15,17 @@ import (
 )
 
 const (
-	pingURL              = "http://unix/" + string(sysconfig.PingModule) + "/ping/"
-	tracerouteURL        = "http://unix/" + string(sysconfig.TracerouteModule) + "/traceroute/"
-	connectionsURL       = "http://unix/" + string(sysconfig.NetworkTracerModule) + "/connections"
-	procStatsURL         = "http://unix/" + string(sysconfig.ProcessModule) + "/stats"
-	registerURL          = "http://unix/" + string(sysconfig.NetworkTracerModule) + "/register"
-	statsURL             = "http://unix/debug/stats"
-	pprofURL             = "http://unix/debug/pprof"
-	languageDetectionURL = "http://unix/" + string(sysconfig.LanguageDetectionModule) + "/detect"
-	netType              = "unix"
+	pingURL                      = "http://unix/" + string(sysconfig.PingModule) + "/ping/"
```

Comment:
maybe abstract `unix` and `localhost:3333` into `hostPort` in the net/common_linux and net/common_windows, and put the other constants in net/common, defined using the value of `hostPort`

Reply:
That is probably out of scope for this PR. I can create a ticket for my team to tackle this refactor.